content/rs/installing-upgrading/upgrading.md

```diff
@@ -58,11 +58,7 @@ $ sudo ./install.sh
 
 During the node upgrade process, the services running RS are restarted.
 This will result in a very short interruption to the proxy and node
-itself and the databases running on it. This can be dramatically reduced
-by using multiple active proxies for your database. The client will lose
-the connection to the proxy on the node being upgraded, but then
-automatically reconnect to another active proxy you have configured for
-the database.
+itself and the databases running on it.
```

Comment:
I think we need to mention here that in case of working with multiple proxies, the interruption will be minimal. 
Gita didn't like the fact that it sounds like we recommend to move to multiple proxies before the upgrade (which makes no sense at all).
Let's try to rephrase it and get Gita's approval